### ISSUE TITLE
(suggestion) Progress bar for duplicate file checks

### ISSUE THREAD

**Issue by pixelb** [pixelb/fslint#17]
Original [issue 17](https://code.google.com/p/fslint/issues/detail?id=17) created by pixelb on 2009-01-12T15:47:00.000Z:

A progress bar of sorts giving an indication how long a search will take
would be a nice improvement, especially when searching large partitions
with many duplicates spread all over the place ;)

**Comment by george-viaud** [comment]
OMG this feature is needed.  Yes, it's impossible to calculate from the start... but if you collect a file list first, and then get a access speed average, you can calculate some flying-average approximation of ETA, or at least a % file completed.  That is, assuming it's done that way :)

**Comment by claell** [comment]
I hope this can be added.

**Comment by vidia** [comment]
I'd also like to throw in support for this feature. I fiddled for a while with the CLI tools before I realized I'd just have to wait.

**Comment by RWL-69** [comment]
I also feel this would be a nice feature but it may be impossible as there are so many variables involved.  

Would it be possible to have a count of files and a percentage of those complete.

At least a status of files or directories being scanned in an abbreviated format on the screen.  It would be an indication that there is activity occurring in the process.

**Comment by snow-abstraction** [comment]
@RWL-69  Yes, I agree. An intermediate and similar feature would be the option to display how many files had been processed by the various phases of the fslint algorithm.

**Comment by hpka** [comment]
Perhaps as an alternative, a verbose output console-like stream, or changing stats that can indicate some activity? Granted I'm suggesting this only to show that the process is still active, but it may provide the reassurance some of us seek.
